riscv-elf/src/rv64.rs

```diff
@@ -0,0 +1,280 @@
+use std::collections::BTreeSet;
+use std::fs;
+use std::path::Path;
+
```

Comment:
This file is similar to the 32-bit case, but way simpler because we're only collecting the labels and jumpdests, no debug info etc.